content/200-orm/200-prisma-client/000-setup-and-configuration/100-custom-model-and-field-names.mdx

```diff
@@ -248,6 +248,13 @@ const userByProfile = await prisma.profile
   .users()
 ```
 
+<Admonition>
```

Comment:
This will work, but you can also use the new Docusaurus style admonitions: https://docusaurus.io/docs/markdown-features/admonitions